src/mbgl/map/source.cpp

```diff
@@ -521,11 +521,16 @@ bool Source::update(MapData& data,
     return allTilesUpdated;
 }
 
-void Source::invalidateTiles(const std::vector<TileID>& ids) {
+void Source::invalidateTiles(const std::unordered_set<TileID, TileID::Hash>& ids) {
```

Comment:
My understanding was `unordered_set` was better when sorting wasn't needed. Is that not the case? Similarly, `unordered_map` vs `map`?